src/command_modules/azure-cli-container/azure/cli/command_modules/container/custom.py

```diff
@@ -288,6 +288,8 @@ def _get_remote_url():
             "A default remote was not found for the current folder. \
             Please run this command in a git repository folder with \
             an 'origin' remote or specify a remote using '--remote-url'")
+    except CalledProcessError:
+        raise CLIError('Command "git remote get-url" not supported in current version of git (git version 2.7.0 or greater required)')
```

Comment:
Instead of coming up with our own text, could you use the exception/error from the ```CalledProcessError``` exception?